src/CsvReader.php

```diff
@@ -138,15 +139,7 @@ protected function readLine()
      */
     protected function validateLineBreak()
     {
-        try {
```

Comment:
`try` tu nema zmysel, kedze `getLineBreak` nevyhadzuje ziadnu exception, je to obycajny getter.

Reply:
jj, i podle coverage to je unreachable